torch/quantization/_numeric_suite.py

```diff
@@ -437,7 +437,7 @@ def compare_model_outputs(
     q_model,
     *data,
     Logger=OutputLogger,
-    white_list=DEFAULT_NUMERIC_SUITE_COMPARE_MODEL_OUTPUT_WHITE_LIST,
+    white_list=DEFAULT_NUMERIC_SUITE_COMPARE_MODEL_OUTPUT_ALLOWED_LIST,
```

Comment:
This needs to be renamed as well
```suggestion
    allow_list=DEFAULT_NUMERIC_SUITE_COMPARE_MODEL_OUTPUT_ALLOWED_LIST,
```